Common/Core/PID/TPCPIDResponse.h

```diff
@@ -89,6 +93,7 @@ class Response
   float mChargeFactor = 2.299999952316284f;
   float mMultNormalization = 11000.;
   bool mUseDefaultResolutionParam = true;
+  float nClNorm = 152.f;
 
   ClassDefNV(Response, 2);
```

Comment:
The ClassDef number should be incremented here for the new data member

Reply:
I set the ClassDef to 3.